examples/run_composer_trainer.py

```diff
@@ -47,7 +47,7 @@ def main() -> None:
     with tempfile.NamedTemporaryFile(mode="x+") as f:
         f.write(hparams.to_yaml())
         trainer.logger.file_artifact(LogLevel.FIT,
-                                     artifact_name="{run_name}/hparams.yaml",
+                                     artifact_name=f"{run_name}/hparams.yaml",
```

Comment:
This looks to be an incorrect fix -- the `artifact_name` is a format string that gets formatted internally. 

For clarity, perhaps the argument should be named `name_format` or something else instead?

Reply:
Hm, so for reference, I see this on my cloud bucket
![image](https://user-images.githubusercontent.com/813367/163862677-c94ee977-d774-4bfb-b3cd-e454e7671fd0.png)
![image](https://user-images.githubusercontent.com/813367/163862750-f4c56e46-07f3-4b9a-bdc7-2aa46c2160cc.png)

so it seems to me that we're missing a format function on the string somewhere -- let me start running jobs and see if I can track down where this is happening

Reply:
Ah the confusion arises because `artifact_name` in `FileLogger` is formattable (https://github.com/mosaicml/composer/blob/d025644e5786a84ac7edebbc6284813df1503c98/composer/loggers/file_logger.py#L82), but in the `logger.file_artifact` method, `artifact_name` is a string.